integration_tests/tests/sql/test_get_column_values_use_default.sql

```diff
@@ -1,4 +1,7 @@
 
+{# This keeps succeeding locally and failing in CI. Disabling it to get everything else out, but it should still be tested. #}
```

Comment:
Let's add a link to the issue in GitHub so that anyone reading will know that there's an issue to address this item of tech debt.

```suggestion
{# This keeps succeeding locally and failing in CI. Disabling it to get everything else out, but it should still be tested.
https://github.com/dbt-labs/dbt-utils/issues/788 #}
```